sources/packages/api/src/services/sfas/sfas-application.service.ts

```diff
@@ -50,6 +50,51 @@ export class SFASApplicationService
     application.extractedAt = getUTC(extractedDate);
     await this.repo.save(application, { reload: false, transaction: false });
   }
+  /**
+   * Validates before an application submission to see if there is an overlapping SFAS fulltime application existing.
+   * @param sin
+   * @param birthDate
+   * @param lastName
+   * @param studyStartDate
+   * @param studyEndDate
+   * @returns SFAS fulltime application.
+   */
+  async validateDateOverlap(
+    sin: string,
+    birthDate: Date,
+    lastName: string,
+    studyStartDate: Date,
+    studyEndDate: Date,
+  ): Promise<SFASApplication> {
+    return this.repo
+      .createQueryBuilder("sfasApplication")
+      .select(["sfasApplication.id"])
+      .innerJoin("sfasApplication.individual", "sfasFTstudent")
+      .where("sfasFTstudent.lastName = :lastName", { lastName })
```

Comment:
Please do the comparison in a not case sensitive way. If I am not wrong the user last name already has a index for lower(lastName).

Reply:
lower added.